src/transaction/mod.rs

```diff
@@ -22,6 +22,7 @@ use crate::{
 };
 use error::TransactionError;
 
+/// Stores the transaction types that are currently supported in starknet.
```

Comment:
I think the words used here can be improved a bit. It doesn't store the tx. Also since it is the mod.rs file of this module we can put a more detailed explanation of what a transaction is and what kind of transactions we have.